releases/release-1.33/release-notes/release-notes-draft.json

```diff
@@ -0,0 +1,1737 @@
+      "bug"
+    ],
+    "sigs": [
+      "cli"
+    ]
+  },
+  "129486": {
+    "commit": "f82439f53654a2f86797c1b949cd337e4e1ba9ce",
+    "text": "Fixed a bug to ensure container-level swap metrics are collected",
+    "markdown": "Fixed a bug to ensure container-level swap metrics are collected ([#129486](https://github.com/kubernetes/kubernetes/pull/129486), [@iholder101](https://github.com/iholder101)) [SIG Node and Testing]",
+    "documentation": [
+      {
+        "description": "[KEP]",
+        "url": "https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2400-node-swap/README.md",
+        "type": "KEP"
+      }
+    ],
+    "author": "iholder101",
+    "author_url": "https://github.com/iholder101",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/129486",
+    "pr_number": 129486,
+    "areas": [
+      "test",
+      "kubelet"
+    ],
+    "kinds": [
+      "bug",
+      "feature"
+    ],
+    "sigs": [
+      "node",
+      "testing"
+    ],
+    "feature": true,
+    "duplicate": true,
+    "duplicate_kind": true,
+    "is_mapped": true
+  },
+  "129500": {
+    "commit": "f1b3fdf7e6d40714b1a43757221832aa1c4a49d1",
+    "text": "Removed general available feature-gate `PDBUnhealthyPodEvictionPolicy`.",
+    "markdown": "Removed general available feature-gate `PDBUnhealthyPodEvictionPolicy`. ([#129500](https://github.com/kubernetes/kubernetes/pull/129500), [@carlory](https://github.com/carlory)) [SIG API Machinery, Apps and Auth]",
+    "author": "carlory",
+    "author_url": "https://github.com/carlory",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/129500",
+    "pr_number": 129500,
+    "areas": [
+      "code-generation"
+    ],
+    "kinds": [
+      "cleanup",
+      "api-change"
+    ],
+    "sigs": [
+      "api-machinery",
+      "apps",
+      "auth"
+    ],
+    "duplicate": true,
+    "duplicate_kind": true
+  },
+  "129506": {
+    "commit": "728a4d2a484e0bf09a6f33e59230246a62d3cc11",
+    "text": "Enabled ratcheting validation on status subresources for CustomResourceDefinitions",
+    "markdown": "Enabled ratcheting validation on status subresources for CustomResourceDefinitions ([#129506](https://github.com/kubernetes/kubernetes/pull/129506), [@JoelSpeed](https://github.com/JoelSpeed))",
+    "author": "JoelSpeed",
+    "author_url": "https://github.com/JoelSpeed",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/129506",
+    "pr_number": 129506,
+    "kinds": [
+      "bug"
+    ],
+    "sigs": [
+      "api-machinery"
+    ],
+    "is_mapped": true
+  },
+  "129543": {
+    "commit": "db1da72beed99f1fcb2955c2624c7dd3531384ea",
+    "text": "DRA API: the maximum number of pods which can use the same ResourceClaim is now 256 instead of 32. Beware that downgrading a cluster where this relaxed limit is in use to Kubernetes 1.32.0 is not supported because 1.32.0 would refuse to update ResourceClaims with more than 32 entries in the status.reservedFor field.",
```

Comment:
```suggestion
    "text": "DRA API: The maximum number of pods that can use the same ResourceClaim is now 256 instead of 32. Downgrading a cluster where this relaxed limit is in use to Kubernetes 1.32.0 is not supported, as version 1.32.0 would refuse to update ResourceClaims with more than 32 entries in the `status.reservedFor` field.",
```